src/seismometer/configuration/config.py

```diff
@@ -286,47 +286,55 @@ def cohorts(self) -> dict:
         """
         return self.usage.cohorts
 
+    @property
+    def comparison_time(self) -> str:
+        """The timestamp to use as reference for comparison across events."""
+        return self.usage.comparison_time
+
     @property
     def events(self) -> dict:
         """
-        List of event objects to use during analysis.
+        Dictionary of all event objects indexed by column name.
 
         Configured in usage_data as events, contains target, outcome, and intervention events with any windowing
         information.
         """
-        return self.usage.events
+        return {ev.display_name: ev for ev in self.usage.events}
 
-    @property
-    def comparison_time(self) -> str:
-        """The timestamp to use as reference for comparison across events."""
-        return self.usage.comparison_time
+    def event_group(self, usage_group: str) -> dict[str, Event]:
+        """
+        Returns a dictionary of events indexed by column name and restricted to the specified usage group
+
+        Configured in usage_data as events with usage 'group'.
+        """
+        return {ev.display_name: ev for ev in self.usage.events if ev.usage == usage_group}
 
     @property
     def targets(self) -> list[str]:
```

Comment:
Should this be updated now? There are several other places where you added the word Dictionary to the return, but the annotation says list still.

Reply:
good catch